R/package.R

```diff
@@ -259,16 +259,12 @@ package_rd <- function(path = ".") {
 }
 
 extract_tag <- function(x, tag) {
-  x %>%
-    purrr::keep(inherits, tag) %>%
-    purrr::map_chr(c(1, 1))
+  purrr::map_chr(purrr::keep(x, inherits, tag), c(1, 1))
```

Comment:
any way to make it clearer than `c(1, 1)`?

Reply:
No, because I don't remember what it does 🤣